tests/unit/components/pacbio/PacbioPoolList.spec.js

```diff
@@ -20,10 +20,6 @@ describe('pacbioPoolList', () => {
     expect(wrapper.find('.pools')).toBeDefined()
   })
 
-  it('contains the correct data', () => {
```

Comment:
Is there a reason this test was removed?

Reply:
Yes I've actually moved this functionality to a new component PacbioPoolSelectedList but I'm realising I haven't tested this separately in there. I'll add a new spec test for that component to add this in instead?

Reply:
OK yeah that sounds sensible

Reply:
I'm now testing this functionality in the e2e test so will be removing the whole test file